packages/auto_mappr/lib/src/builder/map_bodies/class_body_builder.dart

```diff
@@ -158,9 +157,9 @@ class ClassBodyBuilder extends MapBodyBuilderBase {
         final targetField =
             (mapping.target).getAllGetters().firstWhereOrNull((field) => field.displayName == param.displayName);
 
-        final fieldMapping = mapping.tryGetFieldMapping(param.displayName);
+        final fieldMappingX = mapping.tryGetFieldMapping(param.displayName);
```

Comment:
Can we name this variable differently? In outer scope we have already fieldMapping.

Reply:
Fixed, we can use the first fieldMapping